components/GlobalStyle/sub-components/Typography.unit.test.jsx

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
+import { render } from '@testing-library/react';
+
 import Typography from './Typography';
 
 import { expectCSSMatches } from '../test.utils';
 
 describe('Typography component', () => {
   it('Should match snapshot', () => {
-    const typography = renderer.create(<Typography />);
-
-    expect(typography).toMatchSnapshot();
+    expect(render(<Typography />).asFragment()).toMatchSnapshot();
   });
 
   it('Should set standard typeface', () => {
-    renderer.create(<Typography />);
+    render(<Typography />);
 
     expectCSSMatches(`* {
```

Comment:
Did you gave you on replacing these?

Reply:
I don't understand man.

Reply:
Friday, doing the daily, you commented on replacing these `expectCSSMatches`